imperial_coldfront_plugin/templates/imperial_coldfront_plugin/extend_membership.html

```diff
@@ -0,0 +1,9 @@
+{% extends "common/base.html" %}
+{% block content %}
+  <p>Your current membership is due to expire on {{ expiry_date }}.</p>
+  Would you like to extend it?
```

Comment:
Currently looks like this is addressed to the user that is being extended rather than the user doing the extension.